docs/reference/cluster/stats.asciidoc

```diff
@@ -1579,6 +1579,9 @@ The API returns the following response:
    },
    "snapshots": {
      ...
+   },
+   "repositories": {
```

Comment:
is there any other documentation, other than adding the key here?

Reply:
Added a little more docs in 61566556319b.